rplugin/python3/denite/__init__.py

```diff
@@ -9,7 +9,7 @@
 
 if not find_loader('vim'):
     import neovim
-    from denite.ui.default import Default
+    from denite.ui.bridge import Bridge
```

Comment:
Oops. Sorry I should have said that the new implementation will be an official UI while Shougo want to focus to produce more denite specific features and he don't want to pay attention to develop prompt part of the denite.